settings.py

```diff
@@ -240,11 +241,8 @@
 RECAPTCHA_PRIVATE_KEY = ''
 RECAPTCHA_PUBLIC_KEY = ''
 
-CRISPY_TEMPLATE_PACK = 'bootstrap3'
-CRISPY_ALLOWED_TEMPLATE_PACKS = (
-    'bootstrap',
-    'bootstrap3',
-)
+CRISPY_TEMPLATE_PACK = 'bootstrap3to5'
+CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap3to5"
```

Comment:
Previously this was a tuple, now it's a string. Are both of those types supported for this setting?

Reply:
Great question, and something that confused me too. According [to the docs](https://github.com/django-crispy-forms/crispy-bootstrap5/#usage), this is now a string? There's no mention of `CRISPY_ALLOWED_TEMPLATE_PACKS`[ in the read the docs](https://django-crispy-forms.readthedocs.io/en/latest/install.html#template-packs), and the previous functionality to switch template packs that I vaguely remember seems to be completely removed.